articles/custom-domains/index.md

```diff
@@ -164,6 +165,20 @@ If you want to use social identity providers with your custom domain, you must u
 You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
 :::
 
+#### SAML identity provider configuration
+
+If you want to use SAML identity providers with your custom domain, you must obtain the service provider metadata from Auth0 (such as `https://login.northwind.com/samlp/metadata?connection=CONNECTION_NAME`). This includes updated Assertion Consumer Service (ACS) URLs. You will then need to manually update this value in your IdP(s). This change to your IdP(s) will need to happen at the same time as you enable your custom domains. This can pose a problem if there are multiple IdPs to configure.
```

Comment:
> This change to your IdP(s) will need to happen at the same time as you enable your custom domains. This can pose a problem if there are multiple IdPs to configure.

This isn't actually the case. When customers enable custom domains on their tenant, nothing changes until they update their apps to call Auth0 using the custom domain. Apps and connections will keep working with the default tenant domain. So maybe we should change it to something like "This change to your IDP will need to happen if you want to start using the custom domain with your applications."